src/main/java/org/jabref/logic/cleanup/MoveFilesCleanup.java

```diff
@@ -41,8 +48,12 @@ public List<FieldChange> cleanup(BibEntry entry) {
                 if (fileChanged) {
                     changed = true;
                 }
-            } catch (IOException exception) {
-                LOGGER.error("Error while moving file {}", file.getLink(), exception);
+            } catch (FileSystemException exception){
+                LOGGER.warn("Could not move file",exception);
+                dialogService.notify(Localization.lang("Could not move file."));
```

Comment:
Please give a more meaningful message with a reason. As a User I want to know why it could not happen.

Reply:
Is the following message looks good?
Localization.lang("Could not move file. Please close all the attached files and retry.")

Reply:
Now Updated. Please check
![Untitled](https://github.com/JabRef/jabref/assets/71377534/7b44fce7-58c9-4b9b-a6e4-3a0caa954fb2)